extract/extract/extractors/otto.py

```diff
@@ -40,6 +40,9 @@ def extract_otto(parsed_page: ParsedPage) -> Optional[Product]:
     name = properties.get("name", None)
     brand = properties.get("brand", None)
 
+    if brand is None:
+        brand = parsed_page.schema_org.get(JSON_LD, [{}])[0].get("brand", {}).get("name", None)
```

Comment:
`parsed_page.schema_org.get(JSON_LD, [{}])[0]` is exactly what `safely_return_first_element` does, doesn't it?